modules/widgets/simple-payments.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Disable direct access/execution to/of the widget code.
+ */
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! class_exists( 'Jetpack_Simple_Payments_Widget' ) ) {
+	/**
+	 * Simple Payments Button
+	 *
+	 * Display a Simple Payment Button as a Widget.
+	 */
+	class Jetpack_Simple_Payments_Widget extends WP_Widget {
+		/**
+		 * Constructor.
+		 */
+		function __construct() {
+			$widget = array(
+				'classname' => 'simple-payments',
+				'description' => __( 'Add a Simple Payment Button as a Widget.', 'jetpack' ),
+				'customize_selective_refresh' => true,
+			);
+
+			parent::__construct( 'Jetpack_Simple_Payments_Widget', __( 'Simple Payments', 'jetpack' ), $widget );
+		}
+
+		/**
+		 * Front-end display of widget.
+		 *
+		 * @see WP_Widget::widget()
+		 *
+		 * @param array $args     Widget arguments.
+		 * @param array $instance Saved values from database.
+		 */
+		function widget( $args, $instance ) {
+			echo $args['before_widget'];
+
+			$title = apply_filters( 'widget_title', $instance['title'] );
```

Comment:
Could you `/** This filter is documented in core/src/wp-includes/default-widgets.php */` here, to make our parser happy?

Reply:
Added on 5d749a37cc859351c0ee28fb728683d1cdd9d930. Also added the `jetpack_widget_name` filter to the constructor with it's comment.